custom_components/plugwise/config_flow.py

```diff
@@ -310,7 +310,7 @@ def async_get_options_flow(config_entry: ConfigEntry) -> config_entries.OptionsF
 class PlugwiseOptionsFlowHandler(config_entries.OptionsFlow):
     """Plugwise option flow."""
 
-    def __init__(self, config_entry: ConfigEntry) -> FlowResult:
+    def __init__(self, config_entry: ConfigEntry) -> None:
```

Comment:
Why change this to None? The output is of type FlowResult? So the original code is correct.

Reply:
It's not returning anything? (It's the init)

Reply:
Yes, you're right, I looked at the wrong part of the file. Not fully awake yet :)